### ISSUE TITLE
arm32: in_tail: db didn't work

### ISSUE THREAD

**Issue by liutao6869** [fluent/fluent-bit#1893]
## Bug Report

**Describe the bug**

I use Plugin in_tail with db configuration, but fluent-bit always send all file records after restart fluent-bit.

**To Reproduce**
- Steps to reproduce the problem:
1 Compile fluent-bit with 32-bit Arm-Linux system, then install it.
2 Use the fellow configuration:
```
[SERVICE]
    Flush        1
    Daemon       Off
[INPUT]
    Name tail
    Path /usr/local/etc/fluent-bit/test.log
    DB   /usr/local/etc/fluent-bit/test.db
[OUTPUT]
    Name  stdout
    Match *
```
3 Execute fluent-bit
```shell
/usr/local/bin/fluent-bit -c /usr/local/etc/fluent-bit/fluent-bit.conf
``` 
4 Test 
First append some records to log file
```bash shell
#!/usr/bin/env bash
for ((i=1; i<=10; i ++))  
do  
    echo "test $i"  >>  /usr/local/etc/fluent-bit/test.log
done
```
then you will see output in stdout:
```
[0] tail.0: [1579489398.347346952, {"log"=>"test 1"}]
[1] tail.0: [1579489398.347352664, {"log"=>"test 2"}]
[2] tail.0: [1579489398.347353034, {"log"=>"test 3"}]
[3] tail.0: [1579489398.347353319, {"log"=>"test 4"}]
[4] tail.0: [1579489398.347353704, {"log"=>"test 5"}]
[5] tail.0: [1579489398.347353985, {"log"=>"test 6"}]
[6] tail.0: [1579489398.347354231, {"log"=>"test 7"}]
[7] tail.0: [1579489398.347354472, {"log"=>"test 8"}]
[8] tail.0: [1579489398.347354738, {"log"=>"test 9"}]
[9] tail.0: [1579489398.347354983, {"log"=>"test 10"}]
```

Restart fluent-bit:
You will also see the all the records in stdout, like above.

**Expected behavior**

After restart fluent-bit, You should not see any record in stdout.

**Screenshots**

**Your Environment**

* Version used: 
1.3.6
* Configuration:
```
[SERVICE]
    Flush        1
    Daemon       Off
[INPUT]
    Name tail
    Path /usr/local/etc/fluent-bit/test.log
    DB   /usr/local/etc/fluent-bit/test.db
[OUTPUT]
    Name  stdout
    Match *
```
* Environment name and version (e.g. Kubernetes? What version?):
32-bit Arm-Linux Debian 8.

**Comment by liutao6869** [comment]
I debug this with my Arm-Linux board, found it because missing "uint64_t" in [tail_db.c ](https://github.com/fluent/fluent-bit/blob/v1.3.6/plugins/in_tail/tail_db.c#L111). and I create a [new pull request](https://github.com/fluent/fluent-bit/pull/1894) .

**Comment by edsiper** [comment]
thanks for contributing!, the PR is merged and I have just backported the fix to v1.3 branch, the fix will be available in the next v1.3.7 release.

Closing as fixed
